docs/tutorials/downgrade-extension.md

```diff
@@ -51,14 +51,14 @@ spec:
       upgradeConstraintPolicy: SelfCertified
 ```
 
-** Disable CRD Upgrade Safety Check:**
+**Command Example:**
 
-**Patch the ClusterExtension Resource:**
+If you prefer using the command line, you can use `kubectl` to modify the upgrade CRD safety check configuration.
```

Comment:
This is the only content change because it was very odd to read this.

Everything else is just formatting.